birthday/README.md

```diff
@@ -1,92 +1,90 @@
-
-7. `$banUnderAge`
-
-   If set to `true`, the bot will ban users that are under 13 years-old.
-
-# Commands
-
-All commands can be used with bday or birthday. Example: -getbday or -getbirthday<p>
-
-1. **mybirthday 20/12/1998**
-   1. Will set the user birthday to be that date and the bot will congratulate them on that day.<p>
-      Note: syntax is `day/month/year` if you want it to be `month/day/year` set `$invertedOrder` to `true`.
-1. **startbdays**
-   1. Use this command at the time you want the bot to send the birthday msgs.<p>
-      Example: if you use this at 1PM in your local time, the congratulations messages will always be sent at 1PM everyday.<p>
-      Optional duration flag: -startbdays 1h12m<p>
-      Using the command like that would make the bday msg be sent everyday at 1h and 12 minutes after this command was triggered.
-1. **stopbdays**
-   1. This will stop the bdays msgs from being sent.<p>
-1. **setbday**
-   1. This will set a targeted user birthday.<p>
-      Example: -setbday 20/12/1998 @Pedro
-1. **getbday**
-   1. This will tell you the birthday of the specified user.<p>
-      Example: -getbday @Pedro
-1. **delbday**
-   1. This will delete the targeted user birthday. If no user is targeted, it will delete the triggering user bday.
-
-## UPDATING FROM V1.0
-
-I found out that there were a few bugs on this code since I last updated it.<p>
-I **highly** recommend you update your code with this newest version, since all these bugs have been fixed. <p>
-But, one of these bugs were that some users were being congratulated more than once.<p>
-If this is happening on your server, you can run this CC and wait for the bot to send `All set, you can now use V2.0`.<p>
-After that you can update to V2.0.<p>
-If you were not experiencing this issue, you don't need to run this CC.<p>
-| ⚠ This will erase **ALL** birthdays you currently have stored on your server. Users will have to set them up again. |
-| --- |
-
-Code:
+###### Birthday CC trigger
+```
+\A\-(my|start|stop|set|get|del)b(irth)?days?
 ```
+| ℹ Make sure to replace `-` with your server's prefix. |
+| ---- |
+
+Save for now, so it doesn't get lost.
+
+## Configuration
+Before your birthday custom command is ready to go, you still need to configure a few things. Please read carefully through their decription before changing anything.
+
+- `$mods`<br>
+    List of role IDs you consider moderator. Separate multiple role IDs by spaces.
+- `$ChannelID`<br>
+    Configures where the birthday messages are being sent to.
+- `$bdayMsg`<br>
+    This variable defines the message that'll be sent on a member's birthday.
+- `$invertedOrder`<br>
+    Configure which date format will be used. If set to to `true`, YAGPDB will use the american notation `mm/dd/yyyy` instead of `dd/mm/yyyy`.
+- `$yearOptional`<br>
+    Set this to `true` to make the year optional. This will make it default to `2000`, should a member not give a year upon configuring their birthday.
```

Comment:
```suggestion
    Whether the year is optional when configuring birthdays; set to `true` to enable. This will make the year default to `2000`, should a member not give a year upon configuring their birthday.
```